docs/user/configuration/integrations/integrations-next/eventhandler-config.md

```diff
@@ -93,58 +97,8 @@ logs:
         password: YOUR_LOKI_API_KEY
       external_labels:
         cluster: "cloud"
-        job: "integrations/kubernetes/eventhandler"
     positions:
       filename: /tmp/positions0.yaml
-  ## The following stanza is optional and used to configure another client to forward
-  ## Agent's logs to Loki in a K8s environment (using the manifests found below)
-  - name: eventhandler_logs
```

Comment:
Why is this being taken out? Is it no longer needed?

Reply:
Yeah I added it as a convenience for those running `eventhandler` without an Agent logging DS but have since decided it's better to omit to simplify and reduce confusion